src/assets/sass/protocol/components/_card.scss

```diff
@@ -166,3 +166,95 @@
     }
 }
 
+/* -------------------------------------------------------------------------- */
+// A small card with an icon and no link.
+
+.mzp-c-card-picto {
+    color: $color-black;
+    padding: $padding-xl 0;
+    text-align: center;
+
+    .mzp-c-card-picto-content {
+        padding-top: 56px + $padding-lg;
+        position: relative;
+
+        &:before {
+            background: $color-gray-20;
+            content: '';
+            height: 56px;
+            left: 50%;
+            margin-left: -28px;
+            position: absolute;
+            top: 0;
+            width: 56px;
```

Comment:
Not a blocker. 56px is the right spec for the landing page icons and I'll make sure designers stick to that size for the next couple of projects.
But I could see a scenario where we have to support larger icons (similar to what we have on the old homepage https://www.mozilla.org/de/). So we might have to add "mzp-c-card-picto-small", "mzp-c-card-picto-medium", etc. classes at some point.

Reply:
Once we have a need for larger icons we can update 👍